statsmodels/base/tests/test_penalized.py

```diff
@@ -714,8 +714,11 @@ def test_params_table(self):
         # failure for penalized values
         # check only first 2, others fails, see #4669
         exog_index = slice(None, 2, None)
+        exog_index = slice(None, None, None)
```

Comment:
what should a future-reader make of the two versions of `exog_index` here?

Reply:
good question,
the first version of exog_index and comments might now be obsolete.
My guess is that I fixed the underlying problem but had forgotten to clean up the unit test

Reply:
I deleted the old part and extended to comment for the test method.
(When I wrote the test initially I thought TheilGLS sandwich is HC, but it's not)

```
        # override inherited because match is not good except for params and predict
        # The cov_type in GLMPenalized and in TheilGLS are not the same
        # both use sandwiches but TheilGLS sandwich is not HC
        # relative difference in bse up to 7%
```

Reply:
`exog_index = slice(None, None, None)` makes indexing redundant but I keep it for consistency with the super method